lms/djangoapps/instructor/services.py

```diff
@@ -69,3 +71,12 @@ def delete_student_attempt(self, student_identifier, course_id, content_id):
                     )
                 )
                 log.error(err_msg)
+
+    def is_course_staff(self, user, course_id):
```

Comment:
Please use the existing user_has_role(request.user, CourseStaffRole(course_key)) instead of adding another "is_course_staff" implementation.